projects/guidelines.md

```diff
@@ -100,9 +100,10 @@ obtain, provide feedback (as bug reports or enhancements), and contribute to the
 
 * The project's source repository MUST track what changes were made, who made the changes, and when the changes were made.
 * To enable collaborative review, the project's source repository SHOULD include interim versions for review between releases; it SHOULD NOT include only final releases.
-* The project results MUST have a unique version identifier for each release intended to be used by users. It is recommended to use Semantic Versioning.
+* The project results MUST have a unique version identifier for each release intended to be used by users.
+  We recommend to use either [Semantic Versioning](https://semver.org) or [Calendar Versioning](https://calver.org) as appropriate, since popular schemes.
```

Comment:
"since popular schemes" seems unnecessary and it's also broken English.

While I do like the idea of being non-prescriptive, SemVer seems a lot more popular and it communicates important information on version compatibility that is a huge advantage in some contexts (e.g., Julia packages all use SemVer and adding a CalVer package would actually break things - indeed, it would be refused).

So how about:

"We recommend to use the versioning scheme consistent with practices in your sub-domain, e.g. [Semantic Versioning](https://semver.org) or [Calendar Versioning](https://calver.org)."

Reply:
Perfect, much better, thanks!